configdata.pm.in

```diff
@@ -0,0 +1,416 @@
+#! {- $config{HASHBANGPERL} -}
+# -*- mode: perl -*-
+{-
+ sub out_item {
+     my $ref = shift;
+     # Available options:
+     # indent           => callers indentation (int)
+     # delimiters       => 1 if outer delimiters should be added
+     my %opts = @_;
+
+     my $indent = $opts{indent} // 0;
+     # Indentation of the whole structure, where applicable
+     my $nlindent1 = defined $indent ? "\n" . ' ' x $indent : ' ';
```

Comment:
I'm assuming this works, but I'm confused why.  Doesn't line 11 guarantee that $indent is defined?  Should that be a test against zero?  If I'm wrong, can you explain to this noob why?

Reply:
Old code from when it might be undefined, so obvious cleanup needed. Thanks

Reply:
Done